src/vs/workbench/api/common/extHostTypes.ts

```diff
@@ -2001,6 +2001,12 @@ export enum TerminalExitReason {
 	Extension = 4
 }
 
+export enum TerminalShellExecutionCommandLineConfidence {
+	Low = 0,
```

Comment:
could we have comments for these to explain what they mean?

Reply:
These are documented in the actual API, I wanted to avoid duplicating that everywhere